### PR TITLE
Rename answered to recently commented

### LINKED ISSUE #7075: Rename `answered` to `recently_commented`
**Please describe the desired behavior.**
We should consider refactoring the method `answered` to something more accurate, like `recently_commented`, to reflect what the method is actually doing. https://publiclab.org/questions/answered/

**Additional context**

The method named `answered` here is actually getting recently commented questions:
https://github.com/publiclab/plots2/blob/fd5683d47ba235abc38e0e16a0d8675b53a1f289/app/controllers/questions_controller.rb#L94-L107

As seen in this page, with the exception of the index function (showing recently asked questions), the "Recently Commented" tab is the only tab where the url and method name do not match the tab name:
https://github.com/publiclab/plots2/blob/fd5683d47ba235abc38e0e16a0d8675b53a1f289/app/views/questions/index.html.erb#L56-L60

This refactoring may also involve changing the above url from `/questions/answered/` to `/questions/recently-commented`, although I'm not sure how that might affect site navigation or analytics of some kind.

### PR DESCRIPTION
**Pull request by priyanka-choubey**
Fixes #7075 

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below
# GIF 
Hnet-image

### CODE REVIEW

**Comment by priyanka-choubey** [comment]
@jywarren @cesswairimu not really sure why the tests are failing :sweat_smile:

**Comment by Uzay-G** [comment]
Hey @jywarren could you rerun the build. We don't have permissions for this repo but I don't think @priyanka-choubey's changes are making this fail.
Thanks :+1:

File: config/routes.rb [hunk 1]
```diff
@@ -325,10 +325,10 @@
   get 'questions_shadow' => 'questions#index_shadow'
   get 'question' => 'questions#index'
   get 'question_shadow' => 'questions#index_shadow'
-  get 'questions/:author/:date/:id' => 'questions#show' 
+  get 'questions/:author/:date/:id' => 'questions#show'
   get 'questions/show/:id' => 'questions#show'
   get 'q/:id' => 'questions#shortlink'
-  get 'questions/answered(/:tagnames)' => 'questions#answered'
+  get 'questions/recently_commented(/:tagnames)' => 'questions#recently_commented'
```
Comment (jywarren) [inline config/routes.rb]: Hmm. Here, should we preserve the old route, redirecting it to the new route? That way we don't break any existing URLs people have made out there? 

I think it's: `get '/stories', to: redirect('/posts')` for example? 

Thank you!!

**Comment by LeoDog896** [comment]
Looks fine to me! @jywarren's change should be added, but after that this is definitely ready for a merge!

File: config/routes.rb (hunk 1 above)
Comment (priyanka-choubey) [reply config/routes.rb]: i have made the changes.. i have attached a gif showing the redirection happening. Thanx

**Comment by cesswairimu** [comment]
Re-triggering travis

File: config/routes.rb (hunk 1 above)
Comment (jywarren) [reply config/routes.rb:331]: Oh great -- however i don't see the routes changes here... did you push them up? Thanks!

Comment (jywarren) [reply config/routes.rb:331]:
```suggestion
  get 'questions/answered(/:tagnames)', to: redirect('questions#recently_commented')
  get 'questions/recently_commented(/:tagnames)' => 'questions#recently_commented'
```

That should work!

**Comment by jywarren** [comment]
I think i made the backwards-compatible route redirect change!

**Review by jywarren — APPROVED** [review]

**Pull request merged**